docs/_content/standaard/standaard.md

```diff
@@ -100,36 +101,63 @@ Authorization: Bearer eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiIsImNsaWVudF9pZGVudGlmaW
 
 Client en server maken gebruik van `shared secret` om het JWT te signen, met
 het HMAC SHA-256 algoritme. Iedere client MAG een eigen secret hebben. De
-server MOET aan de hand van de `client_identifier` key in de JWT header
+server MOET aan de hand van de `client_id` key in de JWT payload
 de bijhorende secret opvragen. De server MOET met het juiste shared secret
 het JWT valideren tegen tampering.
 
-De ZGW claims in de JWT-payload worden genamespaced onder `zds`.
-
-De claims bevatten de scopes als lijst van strings, waarbij de `scopes` key
-gebruikt wordt.
-
-De `zaaktypes` claim MOET gebruikt worden om zaakgegevens te limiteren. Indien
-deze claim ontbreekt, `null` is of een lege lijst, dan is het VERBODEN om
-zaakgegevens te ontsluiten. Een speciale waarde van `['*']` drukt uit dat alle
-zaaktypes toegestaan zijn.
-
 Voorbeeld van een payload:
 
 ```json
 {
-    "zds": {
-        "scopes": [
-            "zds.scopes.zaken.aanmaken"
-        ],
-        "zaaktypes": [
-            "https://haarlem.ztc.nl/api/v1/zaaktypen/123",
-            "https://haarlem.ztc.nl/api/v1/zaaktypen/124",
-        ]
-    }
+    "iss": "voorbeeld-consumers",
+    "iat": 1556898201,
+    "client_id": "dsh5thqAzL6I9SC5IPgR"
 }
 ```
 
+Na succesvolle validatie van het JWT is nu zeker dat de consumer is wie die
+beweert te zijn.
+
+Providers MOETEN voor elke aanroep door de client na te gaan of de client
+geautoriseerd is om deze aanroep uit te voeren.
+
+Providers MOETEN een geconfigureerde autorisatiecomponent bevragen met het
+`client_id` als query-parameter om de autorisaties van deze client op te
+vragen, conform de API-specificatie van het AC.
+
+Providers MOGEN deze gegevens cachen om performance-redenen. Indien
+er van caching gebruik gemaakt wordt, dan MOETEN providers een mechanisme
+inbouwen om wijzingen in het AC meteen door te voeren in de cache. Het is
+AANGERADEN om hiervoor te abonneren op notificaties verstuurd door het AC.
+
+### Autorisatiecomponent
+
+Alle operaties beschreven in [`openapi.yaml`](../../../api-specificatie/ac/openapi.yaml)
+MOETEN ondersteund worden en tot hetzelfde resultaat leiden als de
+referentie-implementatie van het AC.
+
+Het is NIET TOEGESTAAN om gebruik te maken van operaties die niet beschreven
+staan in deze OAS spec, of om uitbreidingen op operaties in welke vorm dan ook
+toe te voegen.
+
+#### Run-time gedrag
+
```

Comment:
De AC MOET bij de registratie van autorisaties die een of meer zaaktypen bevatten controleren of de zaaktypen bestaan.

Reply:
Waarom moet dat?

Reply:
Ja, dat kan een overweging zijn om het niet te doen. Dan ook vermelden in documentatie.

Reply:
Deze was er inderdaad bewust uitgelaten in eerste opzet, omdat dit betekent dat een AC weer geautoriseerd moet zijn op het ZTC, het ZTC wat zelf tegen dit AC checkt. Ik zou dit open laten, ik formuleer het als 'MAG'.